spec/issues/1061_alternative_time_offsets.spec.js

```diff
@@ -0,0 +1,237 @@
+"use strict";
+
+var Ajv = require("../ajv");
+require("../chai").should();
+
+describe("PR #617, full date format validation should understand leap years", function() {
+  describe("time formats - fast", function() {
+    var ajv;
+    var schema;
+
+    before(function() {
+      ajv = new Ajv({ format: "fast" });
```

Comment:
JSON tests would run for both "full" and "fast" format definitions (I think)

Reply:
Confirmed that the permutations of Ajv instances generated and used by `schema-tests.spec.js` includes explicitly `format: 'full'` and implicitly via default opt `format: 'fast'`.